ui/layout.js

```diff
@@ -64,42 +94,10 @@ class OverviewClone extends St.BoxLayout {
         });
         box.add_actor(searchEntryBin);
 
-        // HACK: invasively find the overview signal id that AppDisplay
-        // will use when connecting to the 'hidden' signal
-        this._overviewHiddenId = Main.overview._nextConnectionId;
-
-        const appDisplayClone = new AppDisplay.AppDisplay();
-
-        appDisplayClone.add_constraint(new Clutter.BindConstraint({
-            source: Main.overview._overview.controls.appDisplay,
-            coordinate: Clutter.BindCoordinate.HEIGHT,
-        }));
-
-        // Disable DnD on clones
-        appDisplayClone._disconnectDnD();
-        appDisplayClone._connectDnD = function() {};
-        appDisplayClone._savePages = function() {};
-
-        // Hide running dots of the clones
```

Comment:
I suppose I could answer this for myself by testing the branch, but: it looks like this change will mean the dots under running apps & pagination dots will be visible on the desaturated clone. Or is there something to give them 0% opacity in the other states of the desktop?

Reply:
Yes, this makes the app grid clone identical to the one in the overview. I should have opened this PR as draft, because there's more stuff to figure out here.